browser/src/slideshow/PresenterConsole.js

```diff
@@ -788,7 +791,8 @@ class PresenterConsole {
 			}
 			case 'next': {
 				this._presenter.getNavigator().dispatchEffect();
-				if (isLastSlide) {
+				// if repeat after sec is set then do not close on last slide
+				if (isLastSlide && !this._presenterInfo.isEndless) {
```

Comment:
Can't we  use `this._presenter._presentationInfo.isEndless`   here?
No sense to add new memeber variable for single use if can be avoided.
It adds possibility to go out of sync with 2 different instances of info when we change `_presenter` in the menatime for example